### PR TITLE
Add char_killlog.md

### PR DESCRIPTION
**Pull request by Celeo**
This endpoint document is a little shaky, though I tried to double-check everything and add notes and references to the bottom of the page. There's some back and forth on whether this page should be used vs. the KillMails endpoint and whether or not this page is a legacy page.

### CODE REVIEW

**Comment by cvweiss** [comment]
The KillLog endpoint documentation should be marked as deprecated. It consists of detrimental behaviors that have been fixed by the KillMail endpoint.

**Comment by Celeo** [comment]
See revised commit. @cvweiss: look good?

File: docs/xmlapi/character/char_killlog.md [hunk 1]
```diff
@@ -1,4 +1,405 @@
 ``` eval_rst 
-.. include:: /stub-notice.rst
+.. warning:: 
+   This article has been marked as deprecated in favor of `Char/KillMails <../character/char_killmails.html>`_.
```
Comment (polarina) [inline docs/xmlapi/character/char_killlog.md]: How about "This _endpoint_ has been marked as deprecated ..."?

**Comment by cvweiss** [comment]
Eh, you might think its a bit extreme but I would leave this page with a deprecation notice only and not include any of the xml documentation at all.

**Comment by bahrmichael** [comment]
I don't agree. Make the deprecation very clear, but leave the decision to the developer. Don't remove documentation that might help in edge cases.

**Comment by cvweiss** [comment]
Then at least include this document I wrote a couple years ago so that the developers can make a very informed decision: https://docs.google.com/document/d/16YfJwjhuH5A3cS4NTMDFDkprnOVKsvgtuRIKk8xjTM8/edit

**Comment by deadlybulb** [comment]
I think it's fair to put a link to this document in the notes section.  The more documentation/discussion the better.

**Comment by Celeo** [comment]
It sounds like the best path is to have the standard contents of the page, with the clear deprecated warning at the top, and the inclusion of several articles, including cvweiss', in the Notes section.

Commit amended.

**Pull request merged**